models/lido/liquidity/linea/lido_liquidity_linea_syncswap_pools.sql

```diff
@@ -0,0 +1,291 @@
+ where token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F
+ union all
+ select 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F
+ ) t
+ )
+
+ , tokens_prices_daily as (     
+select distinct 
+      DATE_TRUNC('day', minute) AS time,
+      t.token AS token,
+      symbol,
+      decimals,
+      AVG(price) AS price
+FROM {{source('prices','usd')}} p
+  JOIN tokens t on p.contract_address = t.token_eth
+{% if not is_incremental() %}
+WHERE DATE_TRUNC('day', p.minute) >= DATE '{{ project_start_date }}'
+{% else %}
+WHERE DATE_TRUNC('day', p.minute) >= DATE_TRUNC('day', NOW() - INTERVAL '1' day)
+{% endif %}
+     and date_trunc('day', minute) < current_date
+     and blockchain = 'ethereum'
+  and contract_address IN (select token_eth from tokens)
+group by 1,2,3,4
+union all
+select distinct
+      DATE_TRUNC('day', minute),
+      t.token AS token,
+      symbol,
+      decimals,      
+      LAST_VALUE(price) OVER (PARTITION BY DATE_TRUNC('day', minute),contract_address  ORDER BY minute NULLS FIRST range BETWEEN UNBOUNDED preceding AND UNBOUNDED following) AS price
+    FROM {{source('prices','usd')}} p
+      JOIN tokens t on p.contract_address = t.token_eth
+    WHERE
+      DATE_TRUNC('day', minute) = current_date
+      and blockchain = 'ethereum'
+  and contract_address IN (select token_eth from tokens) 
+ )
+
+ , wsteth_prices_hourly AS (
+    SELECT distinct
+        DATE_TRUNC('hour', minute) time, 
+        0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F as token,
+        symbol,
+        decimals,
+        last_value(price) over (partition by DATE_TRUNC('hour', minute), contract_address ORDER BY  minute range between unbounded preceding AND unbounded following) AS price
+    FROM {{ source('prices', 'usd') }} p
+    {% if not is_incremental() %}
+    WHERE DATE_TRUNC('day', p.minute) >= DATE '{{ project_start_date }}'
+    {% else %}
+    WHERE DATE_TRUNC('day', p.minute) >= DATE_TRUNC('day', NOW() - INTERVAL '1' day)
+    {% endif %}
+      and blockchain = 'ethereum' and contract_address = 0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0
+      
+    
+) 
+
+, wsteth_prices_hourly_with_lead AS (
+select time, 
+       lead(time, 1, date_trunc('hour', now() + interval '1'  hour)) over (order by time) as next_time, 
+       price
+from wsteth_prices_hourly
+)
+
+, mint_events AS (
+ select DATE_TRUNC('day', m.evt_block_time) AS time,
+      m.contract_address AS pool,
+      cr.token0,
+      cr.token1,
+      SUM(CAST(amount0 AS DOUBLE)) AS amount0,
+      SUM(CAST(amount1 AS DOUBLE)) AS amount1
+ from {{source('syncswap_linea','SyncSwapClassicPool_evt_Mint')}} m
+ left join {{source('syncswap_linea','SyncSwapClassicPoolFactory_evt_PoolCreated')}} cr on m.contract_address = cr.pool 
+ 
+ {% if not is_incremental() %}
+ WHERE DATE_TRUNC('day', m.evt_block_time) >= DATE '{{ project_start_date }}'
+ {% else %}
+ WHERE DATE_TRUNC('day', m.evt_block_time) >= DATE_TRUNC('day', NOW() - INTERVAL '1' day)
```

Comment:
Same as above.

Reply:
Thanks a lot for your suggestions, @Hosuke , I have fixed it accordingly.